sos/report/plugins/vectordev.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class Vectordev(Plugin):
+
+    short_desc = 'A lightweight, ultra-fast tool for building observability pipelines'
+
+    plugin_name = "vectordev"
+    profiles = ('observability',)
+
+    conf_file = "/etc/vector/vector.yaml"
```

Comment:
Did you define this variable to be used in the future? I don't see it used anywhere yet. Perhaps you wanted ot use it in add_copy_spec() below?